src/test/java/swift/logic/commands/ListTaskCommandTest.java

```diff
@@ -27,7 +27,7 @@ public class ListTaskCommandTest {
     public void setUp() {
         model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
         expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
-        expectedModel.updateFilteredPersonList(Model.PREDICATE_HIDE_ALL_PEOPLE);
+        expectedModel.updateFilteredPersonList(Model.PREDICATE_SHOW_ALL_PEOPLE);
```

Comment:
I think its better to show no contacts at first so I think we don't need to change the predicate.